src/api/wayfire/util.hpp

```diff
@@ -148,6 +153,11 @@ class wl_idle_call : public noncopyable_t
     /** Will disconnect if connected */
     ~wl_idle_call();
 
+    wl_idle_call(const wl_idle_call &) = delete;
+    wl_idle_call(wl_idle_call &&) = default;
```

Comment:
Nope, this one isn't trivially movable either. The wayland event loop holds a pointer to this object. Same applies to wl_timer.